windows-vagrant.md

```diff
@@ -1,252 +1,296 @@
 
-2. Run the following two commands in the Command Prompt:
+> **Note**:  this command will require you to download a file that is hundreds of megabytes long.  Make sure you are on a strong Internet connection before you run this command.
 
-   - Run the following command to change your working directory. This will make it so you're working with the web development environment you downloaded earlier.
+Run the following command — and **after the command completes, read the next section before you run another command.**
 
-   `cd ~/Desktop/coding-environment`
+```
+vagrant up
+```
 
-   - Before running the command to turn on your virtual computer environment, understand that the next command will require you to download a file that is hundreds of MBs long. Make sure you are on a strong internet connection before you run this command.
+**NOTE:** If this comes back with an error message telling you to run `vagrant init` DO NOT DO THAT.  See this post [here](faq/vagrant/run-vagrant-init.md) for details about how to fix this error message.
 
-Run the following command **after the previous command completes, read the next section before you run another command.**
 
-`vagrant up`
 
-### Important: Read This Before Running Any Additional Commands
+### IMPORTANT: Read this before running any additional commands.
 
-The previous command may complete with a variety of messages. **It also may instruct you to run commands that will cause problems in your environment**: so make sure to read this section before running anymore commands.
+The previous command may complete with a variety of messages.  **It also may instruct you to run commands that will cause problems in your environment**: so make sure to read this section before doing anything.
 
-**SUCCESS! Everything is successful.**
+**[SUCCESS] — Everything is successful.**
 
 If everything is successful you will see output that looks like the following.
 
-![image alt text](images/windows-vagrant-image_0.png)
-
-If your Command Prompt shows something similar to the above then you can continue to **Step 4**. This means Vagrant has been installed and started as expected.
-
-If this isn't what your screen looks like, continue with the instructions below.
+![Screenshot](images/success-vagrant-windows.png)
 
-The first thing you should do, is review the [ERROR] and [SOMETHING ELSE] messages to find the error that matches the error message you're seeing.
+If you see this, continue to **Step 4**: vagrant has been installed and started as expected.  If this isn't what your screen looks like, continue below and find the message that matches the output you're seeing.
 
-**[ERROR] — Vagrant Environment Or Target Machine Error Message Shown.**
+**[ERROR] — `Vagrant environment or target machine` error message shown.**
 
 The error message will look like this:
 
-![image alt text](images/windows-vagrant-image_1.png)
+![Screenshot](images/vagrant-init-windows.png)
```

Comment:
we should use descriptive alt-text, especially for images conveying information to the user. this will become more and more of a priority with accessibility concerns, so text like "image alt text" or "Screenshot" aren't good enough. The user should be able to get the information they need even if the image doesn't render for some reason.

```suggestion
![Error Message in terminal: A Vagrant environment or target machine is required to run this command...](images/vagrant-init-windows.png)
```